Xamarin.Forms.Platform.Android/Renderers/ShellToolbarTracker.cs

```diff
@@ -451,7 +451,10 @@ protected virtual void UpdateToolbarIconAccessibilityText(Toolbar toolbar, Shell
 			else if (image == null ||
 				toolbar.SetNavigationContentDescription(image) == null)
 			{
-				toolbar.SetNavigationContentDescription(R.String.Ok);
+				if(CanNavigateBack)
+					toolbar.SetNavigationContentDescription(Resource.String.nav_app_bar_navigate_up_description);
+				else
+					toolbar.SetNavigationContentDescription(R.String.Ok);
```

Comment:
should we still set it to ok? maybe we should just not set it to anything? then maybe it'll read like the native default that just reads as "unlabeled button"

Reply:
Is there an option for a "null" or "default"?  Because if I read the code it's not really OK....

Reply:
> Because if I read the code it's not really OK....

What do you mean "read the code?"